### ISSUE TITLE
Teach all recipes flagged as Tier 1 before teaching any flagged as Tier2/Tier3

### ISSUE THREAD

**Issue by barrowisp** [barrowisp/craftastrophe#38]
I like the idea of simply making Crafting Plans choose Fundamental recipes before teaching any others.  But it's more realistic to go the route of allowing Crafting Plans to contain any recipe including advanced ones, and require the player to learn enough before the advanced ones will become added to the recipe book.
#36, #37

**Comment by yooksi** [comment]
Yes, that's why we have the `knowledge` player capability that increases with reading new blueprints.
